raiden-ts/src/transfers/epics.ts

```diff
@@ -68,6 +67,9 @@ import {
   withdrawSendConfirmation,
 } from './actions';
 import { getLocksroot, getSecrethash, makeMessageId } from './utils';
+import { Signer } from 'ethers';
```

Comment:
For consistency, could you please move dependencies imports to the top?

Reply:
Sure, but is there a way to enforce this by a rule with eslint maybe? it seems hard to remember and enforce by hand.

Reply:
Yes we had this rule in another project but I forgot the name

Reply:
@kelsos the rule is `eslint(import/order)` [here](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md)